Gems/PhysX/Code/Editor/EditorJointCommon.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/std/utils.h>
+
+namespace PhysX
+{
+    //! Pair of floating point values for angular limits.
+    using AngleLimitsFloatPair = AZStd::pair<float, float>;
```

Comment:
question: Might it be worth making this a struct that has more useful names like `minExtent`/`maxExtent` or something?

Reply:
i'll see what impact this will have. This was the old type i moved it here to reduce some include overhead.

Reply:
This is used in 40+ locations, so i'm going to leave it for now.

Reply:
Ah okay I see, maybe one for another PR in future, just feels like it would make the code a lot more reasonable - could we potentially add a comment here to indicate what `first` and `second` refer to?